# rGFR : The target force
uint16 r_gfr

# rGWD : The target width
uint16 r_gwd

# rCTR : The control field
uint8 r_ctr